examples/dataflow-scala-kafka2avro/src/main/scala/Object2Kafka.scala

```diff
@@ -0,0 +1,117 @@
+/**
+  Copyright 2019 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  */
+
+package com.google.cloud.pso.kafka2avro
+
+
+import com.google.cloud.pso.kafka2avro.config.Kafka2AvroConfig
+import com.google.cloud.pso.kafka2avro.demo.MyDemoType
+import com.google.cloud.pso.kafka2avro.utils.Kafka2AvroUtils
+import com.spotify.scio.coders.Coder
+import com.spotify.scio.values.SCollection
+import com.spotify.scio.{ ContextAndArgs, ScioContext }
+import org.apache.beam.sdk.io.TFRecordIO.Write
+import org.apache.beam.sdk.io.TextIO.Write
+import org.apache.beam.sdk.io.kafka.KafkaIO
+import org.apache.beam.sdk.transforms.PTransform
+import org.apache.beam.sdk.values.{ PCollection, PDone }
+import org.apache.kafka.common.serialization.StringSerializer
+import org.slf4j.{ Logger, LoggerFactory }
+import pureconfig.error.ConfigReaderFailures
+import pureconfig.generic.auto._
+
+/**
+  * This is just to showcase how to write objects to Kafka. You can use this
+  * Dataflow pipeline to populate Kafka with some objects and test the
+  * Kafka2Avro pipeline
+  */
+object Object2Kafka {
+
+  // T is the type that will be serialized and written to Kafka
+  // Here we only show 1 type, and we use our MyDemotype case class to showcase
+  // how to encode the object into a base64 string
+  type T = MyDemoType
+
+  private val logger: Logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
See comments from the previous pipeline regarding using upper camecase naming for constants